lcli/src/new_testnet.rs

```diff
@@ -145,3 +196,117 @@ pub fn run<T: EthSpec>(testnet_dir_path: PathBuf, matches: &ArgMatches) -> Resul
 
     testnet.write_to_file(testnet_dir_path, overwrite_files)
 }
+
+/// Returns a `BeaconState` with the given validator keypairs embedded into the
+/// genesis state. This allows us to start testnets without having to deposit validators
+/// manually.
+///
+/// The optional `execution_payload_header` allows us to start a network from the bellatrix
+/// fork without the need to transition to altair and bellatrix.
+///
+/// We need to ensure that `eth1_block_hash` is equal to the genesis block hash that is
+/// generated from the execution side `genesis.json`.
+fn initialize_state_with_validators<T: EthSpec>(
+    keypairs: &[(Keypair, Keypair)], // Voting and Withdrawal keypairs
+    genesis_time: u64,
+    eth1_block_hash: Hash256,
+    execution_payload_header: Option<ExecutionPayloadHeader<T>>,
+    spec: &ChainSpec,
+) -> Result<BeaconState<T>, String> {
+    // If no header is provided, then start from a Bellatrix state by default
+    let default_header: ExecutionPayloadHeader<T> =
+        ExecutionPayloadHeader::Merge(ExecutionPayloadHeaderMerge {
+            block_hash: ExecutionBlockHash(eth1_block_hash),
```

Comment:
```suggestion
            block_hash: ExecutionBlockHash::from_root(eth1_block_hash),
```

The second part of a previous suggestion.